modules/transformer-batch/src/test/scala/com/snowplowanalytics/snowplow/rdbloader/transformer/batch/bad/InvalidEnrichedEventsSpec.scala

```diff
@@ -33,8 +33,8 @@ object InvalidEnrichedEventsSpec {
       "failure":{
         "type":"RowDecodingError",
         "errors":[
-          {"type":"InvalidValue","key":"collector_tstamp","value":"29th May 2013 18:04:12","message":"Cannot parse key 'collector_tstamp with value 29th May 2013 18:04:12 into datetime"},
-          {"type":"InvalidValue","key":"event_id","value":"not-a-uuid","message":"Cannot parse key 'event_id with value not-a-uuid into UUID"}
+          {"type":"InvalidValue","key":"collector_tstamp","value":"29th May 2013 18:04:12","message":"Cannot parse key collector_tstamp into datetime"},
```

Comment:
This change makes me very happy!

I know you got this "for free" just by upgrading analytics-sdk, but it's much better.  We should be very careful about every putting customer data into a log message.  This `message` field is now easier to manage from a data-security point of view.